modoptions.lua

```diff
@@ -244,17 +244,14 @@ local options = {
 		key		= "tax_resource_sharing_amount",
 		name	= "Resource Sharing Tax",
 		desc	=	"Taxes resource sharing".."\255\128\128\128".." and overflow (engine TODO:)\n"..
-					"Set to [0] to turn off. Recommened: [0.4]. (Ranges: 0 - 0.99)\n"..
-					"*Disables: Reclaiming of Allied Units, [Unit Sharing] and [Assisting Ally Construction] to prevent loopholes",
+					"Set to [0] to turn off. Recommened: [0.4]. (Ranges: 0 - 0.99)",
```

Comment:
If you're touching the line then may as well:
```suggestion
					"Set to [0] to turn off. Recommended: [0.4]. (Ranges: 0 - 0.99)",
```